app/src/main/Member.js

```diff
@@ -405,6 +409,29 @@ export default class Member extends CoreMember {
         });
     }
 
+    /**
+     * Creates a bulk transfer token builder.
+     *
+     * @param transfers - list of transfers
+     * @param totalAmount - total amount irrespective of currency. Used for redundancy check
+     * @param source - source account for all transfers
+     * @return builder for the token
+     */
+    createBulkTransferTokenBuilder(transfers: Array<BulkTransferBodyTransfers>,  // Array of type BulkTransferBody.transfers
```

Comment:
put it in the @param annotation instead. Also probably should be `BulkTransferBody.Transfer`.

Reply:
I will move the comment up in the annotation section but the second part isn't supported by flow syntax